Assets/MixedRealityToolkit/InputSystem/Pointers/GenericPointer.cs

```diff
@@ -38,7 +39,13 @@ public GenericPointer(string pointerName, IMixedRealityInputSource inputSourcePa
         public string PointerName { get; set; }
 
         /// <inheritdoc />
-        public virtual IMixedRealityInputSource InputSourceParent { get; protected set; }
+        public virtual IMixedRealityInputSource InputSourceParent
```

Comment:
Is there a benefit to not using the auto-property here?

Reply:
Not sure how that got changed. I'll be sure to look at this real quick. Probably a mistake.

Reply:
Yes, it's because it's called in the constructor and because it's virtual can cause issues.

Reply:
https://www.jetbrains.com/help/resharper/2018.1/VirtualMemberCallInConstructor.html